### PR TITLE
Merge upstream into master

### PR DESCRIPTION
**Pull request by nathanshelly**

### CODE REVIEW

File: zsh/zshenv.symlink [hunk 1]
```diff
@@ -1,21 +1,72 @@
...
+
+  export VAULT_ADDR=https://vault.services.opendoor.com:8200
+
+  # output ANSI "color" escape sequences
+  # value of LESS is passed to `less` as args
+  export LESS=" -R"
+  # LESSOPEN configures preprocessor that colors ANSI escape sequences
+  # http://www.andre-simon.de/doku/highlight/en/integration.php
+  export LESSOPEN="| $(command -v highlight) %s --out-format truecolor \
+--line-numbers --force --style base16/onedark --no-trailing-nl"
```
Comment (nathanshelly) [inline zsh/zshenv.symlink]: @sbarbeau-od `less` syntax highlighting using [`highlight`](http://www.andre-simon.de/doku/highlight/en/highlight.php)

Comment (sbarbeau-od) [reply zsh/zshenv.symlink]: the update regarding highlighting in less works in the same terminal after a `make setup`, but it doesn't work if I open a new terminal and I have the following issue for every files:
``` 
less file.py
zsh:1: permission denied: file.py
```

Comment (sbarbeau-od) [reply zsh/zshenv.symlink]: if I manually export LESS and LESSOPEN in my current terminal it works, but not if I open a new terminal 🤔

Comment (nathanshelly) [reply zsh/zshenv.symlink]: 🤦‍♂ sorry should have tested in a new terminal.

What's happening here is that Homebrew isn't added to the path until `path.zsh` runs so `highlight` is not a recognized command in `zshenv`. As a result, `LESSOPEN` gets aliased to the below which when run attempts to call the filename as a command:

```shell
export LESSOPEN="| %s --out-format truecolor \
--line-numbers --force --style base16/onedark --no-trailing-nl"
```

Fixed this by moving `LESSOPEN` to `dynamic_env_vars.zsh` in a862cfb

**Review by sbarbeau-od — APPROVED** [review]
LGTM

**Pull request merged**